demo/pdfpc-demo.tex

```diff
@@ -27,7 +27,8 @@
 \mode<presentation>
 
 \title{\pdfpc demo}
-\author[D. Vilar]{David Vilar \\ \texttt{davvil@gmail.com}}
+\author[D. Vilar]{David Vilar \\ \texttt{davvil@gmail.com} \\
+                  Updated by E. Stambulchik}
```

Comment:
Either put your mail address there as well or remove david's?

Reply:
OK, changed to github hyperlinks. I think nobody wants to get more spam from email web harvesters...